packages/flutter/lib/src/animation/tween.dart

```diff
@@ -17,6 +17,9 @@ export 'curves.dart' show Curve;
 // late Animation<Offset> _animation;
 // late AnimationController _controller;
 
+/// A typedef used to create an [Animatable] from a callback.
```

Comment:
nit: As per style guide link this doc directly to its use by including `[Animatable.fromCallback]` in the doc comment.